lib/resourcebuilder/resourcebuilder.go

```diff
@@ -107,6 +111,18 @@ func (b *builder) Do(ctx context.Context) error {
 				return err
 			}
 		}
+	case *admissionregistrationv1.ValidatingWebhookConfiguration:
```

Comment:
There's [a Python builder][1] to help with these that I'd like to keep up to date (sorry I didn't notice you were touching these soon enough for the generator to help you create this content).  Something like:

```console
$ git --no-pager diff -U1 hack/
diff --git a/hack/generate-lib-resources.py b/hack/generate-lib-resources.py
index 2a45a2ac..a7fa16e8 100755
--- a/hack/generate-lib-resources.py
+++ b/hack/generate-lib-resources.py
@@ -284,2 +284,4 @@ if __name__ == '__main__':
         'github.com/openshift/api/security/v1': {'SecurityContextConstraints'},
+        'github.com/operator-framework/api/pkg/operators/v1': {'OperatorGroup'},
+        'k8s.io/api/admissionregistration/v1': {'ValidatingWebhookConfiguration'},
         'k8s.io/api/apps/v1': {'DaemonSet', 'Deployment'},
@@ -294,2 +296,4 @@ if __name__ == '__main__':
         'github.com/openshift/api/image/v1': {'package': 'github.com/openshift/client-go/image/clientset/versioned/typed/image/v1', 'type': 'ImageV1Client'},
+        'github.com/operator-framework/api/pkg/operators/v1': {'package': 'github.com/operator-framework/operator-lifecycle-manager/pkg/api/client/clientset/versioned/typed/operators/v1', 'type': 'OperatorGroup'},
+        'k8s.io/api/admissionregistration/v1': {'package': 'k8s.io/client-go/kubernetes/typed/admissionregistration/v1', 'type': 'AdmissionregistrationV1Client'},
         'k8s.io/api/apps/v1': {'package': 'k8s.io/client-go/kubernetes/typed/apps/v1', 'type': 'AppsV1Client'},
```

with the OperatorGroup stuff to catch up with #862.

[1]: https://github.com/openshift/cluster-version-operator/blob/964e193242a5a89e9f52ae42278927051e491fee/hack/generate-lib-resources.py#L282-L301

Reply:
Thanks, Trevor! I have added the change for the `OperatorGroup` type in the new commit bdac5ff491a5839370afb2cca793a1e767ab9b4b. And I have updated the `lib: Add ValidatingWebhookConfiguration defaulting` commit (now 58dceccc858962d6a64025d5ffcbe788a55e176c) to include the changes in the helper script, and I have committed the changes generated from the script.